draft-ietf-ivy-network-inventory-yang.md

```diff
@@ -111,64 +111,37 @@ YANG data model that is application- and technology-agnostic.  The
 base data model can be augmented to describe application-specific or
 technology-specific information.
 
-Network Inventory management is a key component in
-operators' OSS architectures.
-
-Network Inventory is a fundamental functionality in network
-management and was specified many years ago.  Given the emergence of
-data models and their deployment in operator's management and control
-systems, the traditional function of inventory management is also
-requested to be defined as a data model.
-
-Network Inventory management and monitoring is a critical
-part for ensuring the network stays healthy, well-planned, and
-functioning in the operator's network.  Network Inventory
-management allows the operator to keep track of which physical
-devices are deployed in the network including relevant software and
+Network inventory is a fundamental functional block in the overall network
+management which was specified many years ago. Network inventory management is a critical
+part for ensuring that the network remains healthy (e.g., auditing to identify faulty elements), well-planned (e.g., identify assets to upgrade or to decommission), and maintained appropriately to meet the performance objectives. Also, network inventory
+management allows operators to keep track of which devices are deployed in their networks, including relevant embedded software and
 hardware versions.
 
-The Network Inventory management also helps the operator to
-know when to acquire new assets and what is needed, or to
-decommission old or faulty ones, which can help to improve network
-performance and capacity planning.
+Exposing standards interfaces to retrieve and query network elements capabilities as recorded in an inventory are key enablers for many applications that consumes network inventory data. From that standpont and given the emergence of standard data models and their deployment by operators, the conventional function of inventory management is also required to be defined as a data model.
```

Comment:
spelling mistake - standpont -> standpoint

Reply:
Thanks @prmanna. Fixed now in https://github.com/ietf-ivy-wg/network-inventory-yang/pull/53/commits/2dcb9076d22c3acbaf004684f3cc2d642b629e66